docs/concepts/reranking.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Reranking"
+---
+
+Vector search or keyword-based functions (like BM25 or ANN) are optimized for speed, not perfect accuracy.
```

Comment:
what is the point you are trying to make here? if its "you can make shit better by calling `rerank()`" I wouldn't do it by saying that bm25/ann are not perfect. that's kind of shitting on ourselves. bm25 doesn't trade recall of speed, ANN does.